java/build.gradle

```diff
@@ -47,11 +47,21 @@ dependencies {
 protobuf {
     generatedFilesBaseDir = "${projectDir}/build/protos/"
     protoc {
-        artifact = "com.google.protobuf:protoc:3.1.0"
+        // for build locally Mac Apple M1
```

Comment:
please remove this before final merging

Reply:
oh, I think that's okay since it beneficial to build on local..? Why need to remove mas?